website/docs/r/storage_object_access_control.html.markdown

```diff
@@ -0,0 +1,144 @@
+---
+# ----------------------------------------------------------------------------
+#
+#     ***     AUTO GENERATED CODE    ***    AUTO GENERATED CODE     ***
+#
+# ----------------------------------------------------------------------------
+#
+#     This file is automatically generated by Magic Modules and manual
+#     changes will be clobbered when the file is regenerated.
+#
+#     Please read more about how to change this file in
+#     .github/CONTRIBUTING.md.
+#
+# ----------------------------------------------------------------------------
+layout: "google"
+page_title: "Google: google_storage_object_access_control"
+sidebar_current: "docs-google-storage-object-access-control"
+description: |-
+  The ObjectAccessControls resources represent the Access Control Lists
+  (ACLs) for objects within Google Cloud Storage.
+---
+
+# google\_storage\_object\_access\_control
+
+The ObjectAccessControls resources represent the Access Control Lists
+(ACLs) for objects within Google Cloud Storage. ACLs let you specify
+who has access to your data and to what extent.
+
+There are two roles that can be assigned to an entity:
+
+READERs can get an object, though the acl property will not be revealed.
+OWNERs are READERs, and they can get the acl property, update an object,
+and call all objectAccessControls methods on the object. The owner of an
+object is always an OWNER.
+For more information, see Access Control, with the caveat that this API
+uses READER and OWNER instead of READ and FULL_CONTROL.
+
+
+To get more information about ObjectAccessControl, see:
+
+* [API documentation](https://cloud.google.com/storage/docs/json_api/v1/objectAccessControls)
+* How-to Guides
+    * [Official Documentation](https://cloud.google.com/storage/docs/access-control/create-manage-lists)
+
+## Example Usage
+
+```hcl
+resource "google_storage_object_access_control" "public_rule" {
+  object = "${google_storage_bucket_object.object.name}"
+  bucket = "${google_storage_bucket.bucket.name}"
+  role   = "READER"
+  entity = "allUsers"
+}
+
+resource "google_storage_bucket" "bucket" {
+	name = "static-content-bucket"
+}
+
+ resource "google_storage_bucket_object" "object" {
+	name   = "public-object"
+	bucket = "${google_storage_bucket.bucket.name}"
+	source = "../static/img/header-logo.jpg"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+
+* `bucket` -
+  (Required)
+  The name of the bucket.
+
+* `entity` -
+  (Required)
+  The entity holding the permission, in one of the following forms:
+    * user-userId
+    * user-email
+    * group-groupId
+    * group-email
+    * domain-domain
+    * project-team-projectId
+    * allUsers
+    * allAuthenticatedUsers
+  Examples:
```

Comment:
This has the same problem as the list above it (you can use the GH "view" button to see how the markdown parses)

Reply:
Oh I didn't even realise that was supposed to be a list. It looks like we can't have content after a list because `lines()` strips it away, so I just inlined those examples.